test/test.c

```diff
@@ -21324,6 +21324,22 @@ void test_SSLHandshakeFirst(void)
     s = natsOptions_TLSHandshakeFirst(opts);
     testCond(s == NATS_OK);
 
+    test("Set TLSHandshakeFirst option without setting secure: ");
+    {
+        // we start with a new natsOptions struct so that we can test
+        // that it does not crash with a minimal config
+        natsOptions *opts = NULL;
```

Comment:
Could we rename this options variable (to avoid shadow)?

Reply:
sure